web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -293,6 +301,8 @@
             on:close={closeViewer}
             on:onVideoEnded={() => (shouldPlayMotionPhoto = false)}
           />
+        {:else if asset?.exifInfo?.projectionType === ProjectionTypeEnum.Equirectangular}
```

Comment:
No need to use an optional chaining for the asset: `asset?.exifInfo?.projectionType` -> `asset.exifInifo?.projectionType`

Reply:
fixed